components/layout/content-slider/content-slider.hbs

```diff
@@ -0,0 +1,193 @@
+<div class="section--bg-white demo-section">
+
+  <div class="content-slider" aria-label="gallery">
+    <hgroup class="featured__title">
```

Comment:
Something to consider, we probably want this to be `content-slider__title`. While the `featured__title` might match the styling we need, we're cherry picking an element out of another block without reference to the parent block in this code.